OptimizelySDK/Optimizely.cs

```diff
@@ -781,5 +782,14 @@ private object GetTypeCastedVariableValue(string value, FeatureVariable.Variable
 
             return result;
         }
-    }
+
+        public void Dispose()
+        {
+            if (Disposed) return;
+
+            Disposed = true;
+            (ProjectConfigManager as IDisposable)?.Dispose();
+            ProjectConfigManager = null;
```

Comment:
If we dispose the optimizely client, but then accidentally end up using it again afterwards, will this result in a null pointer exception?

Reply:
Added test case here. 
https://github.com/optimizely/csharp-sdk/pull/170/files#diff-8660e0a3c8615b111f08b9e4ab4f9b36R3485

Reply:
👍